src/Fixer/PhpUnit/PhpUnitTestCaseStaticMethodCallsFixer.php

```diff
@@ -272,7 +273,13 @@ final class PhpUnitTestCaseStaticMethodCallsFixer extends AbstractPhpUnitFixer i
         'resetCount' => true,
         'stringContains' => true,
         'stringEndsWith' => true,
+        'stringEqualsStringIgnoringLineEndings' => true,
         'stringStartsWith' => true,
+        'assertFileMatchesFormat' => true,
```

Comment:
Line 57 is `// Assert methods`, can we move the assertions to that section, please?

Reply:
I forgot to push the "sort lines" 😮‍💨